src/assets/css/_sass/elements/_page.scss

```diff
@@ -33,4 +31,40 @@ body {
 
 main {
   flex: 1 0 auto; // 3
-}
+  .hero {
+    height: 100vh;
+    background: url('/assets/img/trees.png');
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: $color-text--light;
+    .title__container {
+      display: flex;
+      flex-direction: column;
+      padding-left: 60px; 
+      width: 500px;
+      line-height: normal;
+     .hero__title {
+        @include h1;
+        font-family: Ruda; // Temporary
+        margin: 0;
+        color: $color-text--primary-orange;
+        text-shadow: 2px 2px 1px #1a1a1a;
+        -webkit-margin-after: 0;
+      }
+      .hero__divider {
+        width: 60%;
+      }
+      .hero__date {
+        @include h4;
+        font-family: Ruda;
+        font-weight: 400;
+      }
+      .hero__tagline {
+        font-family: 'Ruda';
+        font-weight: 400;
+      }
+    }
+  }
+}
```

Comment:
set the font on the `.hero` element, the children will inherit it, then set `work sans` on `.button` - but that should be defined in the `.button` elements file

`@include display` will be ruga